sci-mathematics/octave/octave-9.2.0.ebuild

```diff
@@ -0,0 +1,226 @@
+		sci-libs/cxsparse:=
+		sci-libs/umfpack:=
+	)
+	spqr? ( sci-libs/spqr:= )
+	ssl? (
+		dev-libs/openssl:=
+	)
+	sundials? (
+		klu? ( >=sci-libs/sundials-4:=[sparse] )
+		!klu? ( >=sci-libs/sundials-4:= )
+	)
+"
+RDEPEND="${COMMON_DEPS}
+	java? ( >=virtual/jre-1.8:* )"
+DEPEND="${COMMON_DEPS}
+	java? ( >=virtual/jdk-1.8:* )"
+BDEPEND="
+	dev-util/gperf
+	virtual/pkgconfig
+	doc? (
+		dev-texlive/texlive-fontsrecommended
+		dev-texlive/texlive-plaingeneric
+		dev-texlive/texlive-metapost
+		virtual/latex-base
+	)
+	gui? ( dev-qt/qttools:6[linguist] )
+	qrupdate? ( app-misc/pax-utils )
+	sparse? ( app-misc/pax-utils )
+"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-5.1.0-pkgbuilddir.patch
+)
+
+src_prepare() {
+	default
+
+	# occasional fail on install, force regeneration (bug #401189)
+	rm doc/interpreter/contributors.texi || die
+
+	eautoreconf
+}
+
+src_configure() {
+	# libstdc++ bug, bug #887815
+	append-cxxflags -U_GLIBCXX_ASSERTIONS
+
+	# Unfortunate dependency on mpi from hdf5 (bug #302621)
+	use hdf5 && has_version sci-libs/hdf5[mpi] && \
+		export CXX=mpicxx CC=mpicc FC=mpif77 F77=mpif77
+
+	# Some of these use_with flags are a bit mismatched. The configure
+	# script offers only --without-foo, and detects "foo" automatically
+	# unless --without-foo is specified. Passing --with-foo is not an
+	# error, however, so it kind of works. We wind up with, for example,
+	#
+	# --with-sundials_ida (no-op) with USE="sundials"
+	# --without-sundials_ida (disables it) with USE="-sundials"
+	#
+	local myeconfargs=(
+		--localstatedir="${EPREFIX}/var/state/octave"
+		--with-blas="$($(tc-getPKG_CONFIG) --libs blas)"
+		--with-lapack="$($(tc-getPKG_CONFIG) --libs lapack)"
+		--disable-64
+		--enable-shared
+		--with-z
+		--with-bz2
+		--without-fltk
+
+		# bug #901965
+		--without-libiconv-prefix
+		--without-libreadline-prefix
+
+		$(use_enable doc docs)
+		$(use_enable java)
+		$(use_enable json rapidjson)
+		$(use_enable readline)
+		$(use_with curl)
+		$(use_with fftw fftw3)
+		$(use_with fftw fftw3f)
+		$(use_enable fftw fftw-threads)
+		$(use_with glpk)
+		$(use_with hdf5)
+		$(use_with imagemagick magick GraphicsMagick++)
+		$(use_with klu)
+		$(use_with portaudio)
+		$(use_with qhull qhull_r)
+		$(use_with qrupdate)
+		$(use_with sndfile)
+		$(use_with sparse arpack)
+		$(use_with sparse umfpack)
+		$(use_with sparse colamd)
+		$(use_with sparse ccolamd)
+		$(use_with sparse cholmod)
+		$(use_with sparse cxsparse)
+		$(use_with spqr)
+		$(use_with ssl openssl)
+		$(use_with sundials sundials_ida)
+		$(use_with sundials sundials_nvecserial)
+	)
+
+	# Tell autoconf where to find qt binaries, fix bug #837752
+	if use gui ; then
+		export MOC="$(qt6_get_bindir)/../libexec/moc" \
+			UIC="$(qt6_get_bindir)/../libexec/uic" \
+			RCC="$(qt6_get_bindir)/../libexec/rcc" \
+			LRELEASE="$(qt6_get_bindir)/lrelease" \
+			QHELPGENERATOR="$(qt6_get_bindir)/../libexec/qhelpgenerator"
+		myeconfargs+=(
+			$(use_with gui opengl)
+			$(use_with gui x)
+			"--with-qt=6"
```

Comment:
`"--with-qt=6"` and this can be `$(use_with gui qt 6).

According to [1], the third argument will be put after the `=`.


[1] https://devmanual.gentoo.org/function-reference/query-functions/index.html